modules/telco-core-installation.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-core-ref-components.adoc
+
+:_content-type: REFERENCE
+[id="telco-core-installation_{context}"]
+= Installation
+
+New in this release::
```

Comment:
Is this section needed?

Reply:
It is in original and I believe it is needed.